pkg/client/config/config.go

```diff
@@ -29,21 +29,22 @@ import (
 )
 
 var (
-	kubeconfig, masterURL string
-	log                   = logf.RuntimeLog.WithName("client").WithName("config")
+	kubeconfig, controlPlaneURL string
```

Comment:
switch to `apiServerAddress` as the variable name for the deprecation period?

Reply:
I'm fine either way, @DirectXMan12 ?

Reply:
yeah, use `apiServerURL` as the variable name sot that we're clear.